libs/remix-ui/run-tab/src/lib/components/mainnet.tsx

```diff
@@ -2,6 +2,7 @@
 import React, { useEffect, useState } from 'react'
 import { CopyToClipboard } from '@remix-ui/clipboard'
 import Web3 from 'web3'
+import { fromWei, toBN, toWei } from 'web3-utils-legacy'
```

Comment:
This was a temporary decision made to reduce the friction of migrating to `4.x` since `BN` is no longer used

Reply:
I've removed this dependency